scripts/run_tests.sh

```diff
@@ -26,8 +26,8 @@ show_coverage(){
 
 run_test(){
```

Comment:
Is it possible to automate the local install and removal when doing `bash scripts/run_tests.sh all`?

The current setup requires developers to manually install and remove the package. Prefer to maintain the original workflow as it may be necessary to run the full test suite locally.

Reply:
This is a good idea, but let me think through this and do some research, due to:
1. `pip install .` will overwrite existing Peeking Duck installs in current environment, so not good for the user.
2. Ideally we create and install into a new test environment, but we will have to deal with different installation procedures/packages/pythons for different OS. Non-trivial to get this correct cross-platform.

Reply:
As per discussed, will leave `run_tests.sh` unchanged and do the necessary install and tests in the post merge script instead.